src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -292,6 +328,18 @@ private async Task WriteForAllLocalDatabases(ZipArchive archive, JsonOperationCo
                         await WriteDatabaseInfo(archive, jsonOperationContext, localEndpointClient, databaseName, token);
                     }
                 }
+
+                HashSet<string> GetDatabases()
+                {
+                    var allDatabases = ServerStore.Cluster.GetDatabaseNames(context);
+
+                    if (string.IsNullOrEmpty(databases) == false && databases.Any())
```

Comment:
this should be `databases.Count > 0`, it is a struct

Reply:
done